### ISSUE TITLE
Suggestion: show new episodes available for download / 2 more columns

### ISSUE THREAD

**Issue by M-A-D-A-O** [erengy/taiga#235]
the idea is to highlight the anime if there are episodes for download that are not in your library folders and/or maybe add 2 columns: one for episodes in your library folder and another for aired eps.
i know that you can get that info out of the "Progress" column (in your anime list), but sometimes its hard to see there, especially with quite long anime (100+ eps).

**Comment by saulens22** [comment]
Good idea. I would suggest using different highlight color (like yellow) to highlight not-complete series, and if there are episodes for download, it might show it in 'Now Playing' tab (it's kind of empty now).
